#exception: python object that represents a possible failure condition
# few standard exceptions: File not found, import errors, runtime errors, name error, syntax errors

#try :detects exception
#except: catches exception

try:
     a=10
     b=20
     c=a/b
     print("the answer of a/b is:" , c)

except:
    print("cannot divide by zero.enter valid input")

try:
    a=int(input("firstnumber"))
    b=int(input("second number"))
    c=a/b
    print("answer of a/b is:" ,c)

except ValueError:
    print("enter valid input")

except ZeroDivisionError:
    print("Cannot divide by zero")

#except(ValueError, ZeroDivisionError):
    #print("enter valid input")

finally:
    print("Loop over")


